bundle/src/main/java/com/adobe/acs/commons/util/ResourceServiceManager.java

```diff
@@ -38,6 +38,8 @@
 import org.osgi.framework.ServiceReference;
 import org.osgi.framework.ServiceRegistration;
 import org.osgi.service.component.ComponentContext;
+import org.osgi.service.component.annotations.Activate;
```

Comment:
Because this is a public abstract class, I recommend leaving the SCR annotation in place here, in addition to the DS annotation, so that consumers using Felix SCR tooling will still find the old annotation in the class file.

Reply:
@adamcin Are you saying it should have both/2 @Activate annotations? One from felix.scr and other from osgi.annotations?

Curious - what Felix SCR tooling might someone still be using today?

Reply:
Yes, exactly. As in:

```
@Activate
@org.apache.felix.scr.annotations.Activate
protected void activate() {
}
```

Any customers with components that extend from these classes without overriding the annotated method would still have to be using SCR annotations. (Using DS component inheritance is discouraged, but it is supported by default with maven-bundle-plugin when using SCR annotations).

Reply:
I wouldn’t consider such edge cases. Too much maintenance overhead and easy to fix on consumer side.